__tests__/useCko.spec.ts

```diff
@@ -64,6 +88,27 @@ jest.mock('../src/payment', () => ({
   }))
 }));
 
+jest.mock('@vue-storefront/core', () => {
+  const refsMap = new Map();
+  return { 
+    sharedRef: (value, key) => {
```

Comment:
why? what if we change the implementation of sharedRef? this one should be fully mocked